src/agenda.rs

```diff
@@ -432,6 +470,22 @@ pub fn prioritization<'a>() -> Box<dyn Action> {
             filters: vec![("state", "open")],
             include_labels: vec!["I-nominated", "T-compiler"],
             exclude_labels: vec![],
+            ordering: HashMap::new(),
+        },
+    });
+
+    let mut ordering = HashMap::new();
+    ordering.insert("sort", "updated");
+    ordering.insert("per_page", "5");
+
+    queries.push(QueryMap {
+        name: "top_unreviewed_prs",
+        query: github::Query {
+            kind: github::QueryKind::List,
+            filters: vec![("state", "open"), ("is", "pull-request"), ("-is", "draft")],
```

Comment:
```suggestion
            filters: vec![("state", "open"), ("is", "pull-request"), ("draft", "false")],
```

@apiraino according to your recent review this should be `draft: false`? is this correct?.